regression/multi_regression.sh

```diff
@@ -1,22 +1,18 @@
 #!/bin/sh --login
 
-regtests_all="global_3dvar
-              global_4dvar
-              global_4denvar
+regtests_all="global_4dendvar
```

Comment:
`global_4dendvar` should read `global_4denvar`.   Please remove the `d` between `4den` and `var`.

Reply:
Corrected at [549497b](https://github.com/NOAA-EMC/GSI/pull/646/commits/549497b7c9648c77c6897f41ff7b92e7af613fa7).